### PR TITLE
[DO NOT MERGE] test e2e-aws failures

### PR DESCRIPTION
**Pull request by umohnani8**
Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

```release-note
none
```

### CODE REVIEW

File: hack/build-rpms.sh [hunk 1]
```diff
@@ -28,6 +28,11 @@ tar czf "${rpm_tmp_dir}/SOURCES/${OS_RPM_NAME}-test.tar.gz" \
     .
 cp -r "${ci_data}/." "${rpm_tmp_dir}/SOURCES"
 
+# for file in /etc/yum.repos.d/*; do sed "s#vault.centos.org/centos/7#vault.centos.org/centos/7.8.2003#g" $file -i; done
+echo $releasever
```
Comment (fgiudici) [inline hack/build-rpms.sh:32]: From the error in the CI job seems the script is run with set -u
```suggestion
echo ${releasever-""}
```

**Comment by fidencio** [comment]
JTFR, I've opened https://pagure.io/centos-infra/issue/156 to track the status of the issue on CentOS Infra side.

**Comment by fidencio** [comment]
Let me quite a comment from Fabian, in the issue mentioned above:
```
Hi,

Some  just reported the same issue on centos-devel list, and it seems that @hughesjr indeed forgot to update metadata. (and also some real src.rpm should also have landed there, probably still in his todo list for post-GA activities)

I've created empty repodata for now, so that shouldn't "block" you but I'll let this ticket assigned so that he can confirm and close it when properly implemented
```

I've re-triggered the CI on #4356 and it seems to have passed the problematic part.  So, we should be in a good shape about this issue now.

**Comment by fidencio** [comment]
@umohnani8, we can drop this patch as the workaround provided by Fabian did fix the issues with CentOS. \o/.

And now we know the proper way to poke CentOS Infra (via a ticket) in case this happens again in the future.

**Comment by umohnani8** [comment]
Yup! I was just trying to see if updating the version helps. Thanks for the update! Just saw that is has been fixed on the centos end, so closing.

**Pull request closed without merging**